slices/apache2.yaml

```diff
@@ -0,0 +1,166 @@
+package: apache2
+
+essential:
+  - apache2_copyright
+
+slices:
+  bins:
+    essential:
+      - apache2-bin_bins
+      - apache2_config
+      - apache2-utils_bins
+      - base-files_bin
+      - base-files_var
+      - coreutils_bins
+      - media-types_data
+      - perl-base_bins
+      - perl-base_modules
+      - perl-modules-5.38_core
+      - perl-modules-5.38_system-utils
+      - procps_bins
+    contents:
+      /usr/sbin/a2disconf:
+      /usr/sbin/a2dismod:
+      /usr/sbin/a2dissite:
+      /usr/sbin/a2enconf:
+      /usr/sbin/a2enmod:
+      /usr/sbin/a2ensite:
+      /usr/sbin/a2query:
+      /usr/sbin/apache2ctl:
+      /usr/sbin/apachectl:
+
+  config:
+    essential:
+      - apache2-data_data
+      - apache2_documentroot
+    contents:
+      /etc/apache2/apache2.conf:
+      /etc/apache2/conf-available/**:
+      /etc/apache2/conf-enabled/charset.conf:
+        symlink: /etc/apache2/conf-available/charset.conf
+      /etc/apache2/conf-enabled/localized-error-pages.conf:
+        symlink: /etc/apache2/conf-available/localized-error-pages.conf
+      /etc/apache2/conf-enabled/other-vhosts-access-log.conf:
+        symlink: /etc/apache2/conf-available/other-vhosts-access-log.conf
+      /etc/apache2/conf-enabled/security.conf:
+        symlink: /etc/apache2/conf-available/security.conf
+      /etc/apache2/conf-enabled/serve-cgi-bin.conf:
+        symlink: /etc/apache2/conf-available/serve-cgi-bin.conf
+      /etc/apache2/envvars:
+      /etc/apache2/magic:
+      /etc/apache2/mods-available/**:
+      /etc/apache2/mods-enabled/access_compat.load:
+        symlink: /etc/apache2/mods-available/access_compat.load
+      /etc/apache2/mods-enabled/alias.conf:
+        symlink: /etc/apache2/mods-available/alias.conf
+      /etc/apache2/mods-enabled/alias.load:
+        symlink: /etc/apache2/mods-available/alias.load
+      /etc/apache2/mods-enabled/auth_basic.load:
+        symlink: /etc/apache2/mods-available/auth_basic.load
+      /etc/apache2/mods-enabled/authz_core.load:
+        symlink: /etc/apache2/mods-available/authz_core.load
+      /etc/apache2/mods-enabled/authn_file.load:
+        symlink: /etc/apache2/mods-available/authn_file.load
+      /etc/apache2/mods-enabled/authz_host.load:
+        symlink: /etc/apache2/mods-available/authz_host.load
```

Comment:
The YAML linter states the `contents` list isn't sorted correctly.
I think it is because `authn_file.load ` should be before `authz_core.load `, like this:
```
      /etc/apache2/mods-enabled/authn_file.load:
        symlink: /etc/apache2/mods-available/authn_file.load
      /etc/apache2/mods-enabled/authz_core.load:
        symlink: /etc/apache2/mods-available/authz_core.load
      /etc/apache2/mods-enabled/authz_host.load:
        symlink: /etc/apache2/mods-available/authz_host.load
```